.travis.yml

```diff
@@ -1,8 +1,8 @@
 sudo:     required
 language: go
 go:
-  - "1.12.x"
   - "1.13.x"
+  - "1.14.x"
```

Comment:
nit perhaps 1.14.x and 1.15.x since those are the supported versions now?

Reply:
Just added 1.15.X. Gonna keep 1.13.X for now since its still being used and I don't wanna accidentally introduce a breaking change. Will remove 1.13.X once its not so widely used.